submissions/nwokejeme-immaculeta/html-forms/homepage.html

```diff
@@ -0,0 +1,123 @@
+<html>
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF=8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>IMMA CLASSIQUE</title>
+    <link rel="stylesheet" href="../css/homepage.css">
+</head>
+<body>
+    <header>
+        <div class="navbar">
+            <div id="navbranding">
+                <h2>IMMA CLASSIQUE</h2>
+                <p>...heart of best designs </p>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="../html/product.html">Product Page</a></li>
+                    <li><a href="../html-forms/contactpage.html">Contact Us</a></li>
+                    <li><a href="../html-forms/homepage.html">Sign-up</a></li> 
+                </ul>
+            </nav>
+        </div> 
+    </header>
+    <section class="container">
+        <main><br> <br>
+            <h1>Register your Account</h1> 
+            <p>Registering your account gives you access to sweet discount, promo and events available only to registered customers. </p> <br> <br>
+           <p>Already got an account? <a href="#">Sign in</a></p>
+        </main>
+        <aside>
+            <form id=validateForm action="#">
+                <div class="form-details">
+                    <label for="firstName">First Name</label>
+                    <br>
+                    <input type="text" name="firstName" id="firstName" maxlength="50">
```

Comment:
There is no need to add text limit to your input field. That should come in when validating the form @Immaculeta1